ex1/exceptions.py

```diff
@@ -38,6 +38,28 @@
 # also named search_n. The function should do the same as functions.search_n
 # but if the variable is not found in the list then raise a ValueError.
 
+def search_n(l,x):
+    """ Search for the first x in list l and returns a tuple with index and value.
+    	It only returns the first found value. If no value was found a ValueError Exception will raise.
+    	
+    Parameters
+    ----------
+    l: list
+    x: any
+    
+    Returns
+    -------
+    List
+    
+    Examples
+    --------
+    >>>search_n([1,2,3,4,5,3,6],3)
+    (2,3)
+    """
+    idx = l.index(x)
+    return (idx,l[idx])
```

Comment:
This is arguably more elegant than what I asked for. But the main point was to work with functions that have the same name but are in different namespaces.